terraform/modules/aws-brig-prekey-storage-event-queue/README.md

```diff
@@ -0,0 +1,18 @@
+Terraform module: Brig pre-key storage & event queue 
+===================================================
+
+This module creates AWS resources (SQS, DynamoDB) that allows Brig to store pre-keys on AWS and establishes a message
```

Comment:
Prekeys are not stored on DynamoDB. DynamoDB is purely used as a distributed locking mechanism to avoid race conditions on prekey insertion and retrieval. All storage happens in cassandra. Please change the module name, the header in this file, and the line above to reflect that (and remove the word "storage").

Reply:
done. please confirm whether this is suitable now.